include/swift/AST/TypeRepr.h

```diff
@@ -1101,6 +1101,44 @@ class OpaqueReturnTypeRepr : public TypeRepr {
   friend class TypeRepr;
 };
 
+/// A TypeRepr for a type with a generic parameter list of named opaque return
+/// types.
+///
+/// This can occur only as the return type of a function declaration, to specify
```

Comment:
It can also occur as the return type of a computed property or subscript, no? (Those should be tested as well)